assets/fonts/OpenSans/font-face.css

```diff
@@ -0,0 +1,11 @@
+@font-face {
```

Comment:
Sure if that has the necessary weight, PT Sans has only 400 (nornal) and 700 (bold). Open Sans that I added has 300 (light)